lighthouse/helpers/plates.py

```diff
@@ -693,17 +698,17 @@ def __sample_subject_for_dart_control_row(dart_control_row: Dict[str, str]) -> D
     }
 
 
-def format_plate(barcode: str) -> Dict[str, Union[str, bool, Optional[int]]]:
-    """Used by flask route /plates to format each plate. Determines whether there is sample data for the barcode and if
-    so, how many samples meet the fit to pick rules.
+def render_fields_format_plate(barcode: str) -> Dict[str, Callable[[], Union[str, bool, SampleDocs, Optional[int]]]]:
```

Comment:
I find this name confusing/hard to read, what about `plate_field_wrapper`, `plate_fields`, `fields_for_plate`?

Reply:
Ok, going with field_generators_for_plate_lookup as I am thinking they are the fields+values